src/app/directives/directives.module.ts

```diff
@@ -25,23 +25,23 @@
 
 import { NgModule } from '@angular/core';
 import { DocumentListDirective } from './document-list.directive';
-import { PaginationDirective } from './pagination.directive';
 import { LibraryMembershipDirective } from './library-membership.directive';
 import { LibraryFavoriteDirective } from './library-favorite.directive';
 import { LockNodeDirective } from './lock-node.directive';
+import { SharedModule } from '@alfresco/aca-shared';
 
 export function directives() {
   return [
     DocumentListDirective,
-    PaginationDirective,
     LibraryMembershipDirective,
     LibraryFavoriteDirective,
     LockNodeDirective
   ];
 }
 
 @NgModule({
+  imports: [SharedModule],
   declarations: directives(),
-  exports: directives()
+  exports: [directives(), SharedModule]
```

Comment:
why directives is exporting Shared? should not it be the opposite?

Reply:
same as above